feedback/static/html/staff_item.html

```diff
@@ -0,0 +1,15 @@
+<div class="feedback_likert_rating">
+  <label title="{scale_text}" class="feedback_likert_label">
+    <input id="radio_{idx}" name="feedback_scale" class="feedback_radio" type="radio" {active}/>
```

Comment:
For accessibility, you have to group the radio buttons with the same name together. See
https://dequeuniversity.com/rules/axe/1.1/radiogroup

FYI @cptvitamin